src/components/Alert/README.md

```diff
@@ -0,0 +1,234 @@
+<ExampleBlock
+    code={`
+<Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>} />
+<Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>} />
+`}>
+    <UIKit.Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<UIKit.Button>button</UIKit.Button>} />
+    <UIKit.Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<UIKit.Button>button</UIKit.Button>} />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert layout="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>}/>
+<Alert layout="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>}/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Corners
+
+`rounded` - used fot round corners of alert window (used by default)
+
+`square` - used for square corners of alert window.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+<Alert title="Square" message="Square corners" corners="square" />
+`}
+>
+    <UIKit.Alert title="Rounded" message="Rounded corners" corners="rounded"  />
+    <UIKit.Alert title="Square" message="Square corners" corners="square" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title="Rounded" message="Rounded corners" corners="rounded"/>
+<Alert title="Square" message="Square corners" corners="square"/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert title
+
+`title` - title of an alert. Override if by Alert.Title component.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+`}
+>
+    <UIKit.Alert title={<UIKit.Alert.Title className={'some-class'} text="some text"></UIKit.Alert.Title>} />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert title={<Alert.Title className={'some-class'} text="some text"></Alert.Title>} />
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Alert message
+
+`message` - The message of an alert should completely explain the alert's content.
+
+## Alert onClose
+
+`onClose` - enabled close button and defined function in click to this button.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Alert onClose={() => alert('Close button pressed')} title="Alert has close" message="Alert has close" />
+`}
+>
+    <UIKit.Alert onClose={() => alert('Close button pressed')} title="Alert has close" message="Alert has close" />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert
+  onClose={() => alert('Close button pressed')}
+  title="Alert has close"
+  message="Alert has close"
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Align
+
+`baseline` - align used bu default.
+
+`center` - centered all content in vertical direction, useful if for some reason your actions take more space than text
```

Comment:
`centered all content in vertical direction` -> `the content is vertically centered within the Alert component`